tests/Feature/Api/ApplicantSkillsControllerTest.php

```diff
@@ -24,7 +24,7 @@ public function testGetApplicantSkills(): void
         $response = $this->actingAs($applicant->user)->json('get', route('api.v1.applicant.skills.index', $applicant));
         $response->assertOk();
         $response->assertJson([
-            'skill_ids' => $skills->sortBy('id')->pluck('id')->all()
+            'skill_ids' => $skills->sort()->pluck('id')->all()
```

Comment:
Any reason it was changed _to_ sortBy on line 19 and _away from_ sortBy on line 27?

Reply:
My thinking was that the collection in $skills on line 18 contains an array with items that each have an `id` key, whereas on line 27, $skills is only an array of the values of the `id` key and therefore there would be no `id` key to sort by, but I believe I am wrong since it's only an array of the values of the `id` key _after_ the `sortBy` method is called, since the `pluck` method is called after. My fault.